### ISSUE TITLE
[MAS2.4.7] [Keyboard Navigation-Open Bot Dialog] Visual focus indicator is not clearly visible for “Learn More” link available at "Open Bot" dialog.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1954]
**Actual Result:**
Visual focus indicator is not clearly visible for “Learn More” link available at "Open Bot" dialog.

**Expected Result:**
Visual focus indicator should be clearly visible for “Learn more” links

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the Welcome tab and reach at "Open Bot" button and activate it.
4. "Open a Bot" dialog gets appeared, navigate to "Learn More" link.
5. Observe the issue.

**Note:**
1. Same issue observed on “Learn More” link available at “Create new bot configuration” pop-up.
2. Same issue found in “Add an Endpoint for your bot” dialog box.
3. Same issue observed at “Privacy Statement” link available at settings pane.

**User Impact:**
Users who rely on Keyboard navigation will be affected the keyboard focus indicator is not clearly visible on control so that keyboard dependant users will be affected.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B3b98b04f-4fab-4d3e-81af-a0906a34849c%7D&cid=46771981-36ee-48c9-a275-2a5883351563](url)

**Attachments:**
[Visual focus is not visible on Learn More link.zip](https://github.com/microsoft/BotFramework-Emulator/files/3794112/Visual.focus.is.not.visible.on.Learn.More.link.zip)

**Comment by tonyanziano** [comment]
This has been fixed and is ready to be verified in the latest nightly version:

![image](https://user-images.githubusercontent.com/3452012/70383645-1d585000-1926-11ea-9e9e-2a0f32899ce2.png)

**Comment by ShikhaMishra11** [comment]
Thanks @tonyanziano We have also verified in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0.94518
